pkg/authenticator/username_test.go

```diff
@@ -0,0 +1,24 @@
+package authenticator
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestUsername(t *testing.T) {
+	Convey("NewUsername", t, func() {
```

Comment:
I think we should elaborate the testing here, and to understand which behavior we are expecting, for example:
1. username = string1
2. username = string1/string2

Reply:
thanks, added more tests